### PR TITLE
GLIDENumber update

### PR DESCRIPTION
**Pull request by codeforIATIbot**
Codelist update, sent from [this TravisCI build](https://travis-ci.com/codeforIATI/codelist-updater/builds/174180404).

### CODE REVIEW

File: xml/GLIDENumber.xml [hunk 1]
```diff
@@ -25,6 +25,19 @@
...
+            <code>FF-2020-000158-SRB</code>
+            <codeforiati:event-code>FF</codeforiati:event-code>
+            <codeforiati:event>
+                <narrative>Flash Flood</narrative>
+            </codeforiati:event>
+            <codeforiati:country-code>SRB</codeforiati:country-code>
+            <codeforiati:country>
+                <narrative>Serbia</narrative>
+            </codeforiati:country>
+            <codeforiati:date>2020-07-22</codeforiati:date>
```
Comment (andylolz) [inline xml/GLIDENumber.xml:38]: This date is in the future 🤔

Comment (andylolz) [reply xml/GLIDENumber.xml:38]: Looks like a mistake at source: https://glidenumber.net/glide/public/search/details.jsp?glide=22016&record=1&last=1

**Comment by markbrough** [comment]
Yeah, that field appears to be a free-text field so it's often a bit dodgy - I think we should go ahead and accept this?

**Comment by andylolz** [comment]
> I think we should go ahead and accept this

Yeah, that sounds fine. I’ve amended the commit message, just to make a note of it there.

**Review by andylolz — APPROVED** [review]

**Review by andylolz — APPROVED** [review]

**Pull request merged**